nautobot/circuits/filters.py

```diff
@@ -102,24 +98,13 @@ def search(self, queryset, name, value):
         ).distinct()
 
 
-class CircuitTypeFilterSet(
-    BaseFilterSet,
-    CustomFieldModelFilterSet,
-    NameSlugSearchFilterSet,
-    CreatedUpdatedFilterSet,
-):
+class CircuitTypeFilterSet(NautobotFilterSet, NameSlugSearchFilterSet):
     class Meta:
         model = CircuitType
         fields = ["id", "name", "slug"]
 
 
-class CircuitFilterSet(
-    BaseFilterSet,
-    StatusModelFilterSetMixin,
-    CustomFieldModelFilterSet,
-    TenancyFilterSet,
-    CreatedUpdatedFilterSet,
-):
+class CircuitFilterSet(NautobotFilterSet, NameSlugSearchFilterSet, StatusModelFilterSetMixin, TenancyFilterSet):
```

Comment:
Was the addition of `NameSlugSearchFilterSet` here intentional?

Reply:
Nope! I did this on the train earlier and it was a bit bumpy! Will sort it in the morning!